### PR TITLE
MINOR: Fix Datetime Conversion issue in usage/lineage

### PR DESCRIPTION
**Pull request by SumanMaharana**
### Describe your changes:

Fix Datetime Conversion issue in usage/lineage

#
### Type of change:

- [x] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/database/mssql/usage.py [hunk 1]
```diff
@@ -37,32 +25,3 @@ class MssqlUsageSource(MssqlQueryParserSource, UsageSource):
         AND lower(t.text) NOT LIKE '%%create%%function%%'
         AND lower(t.text) NOT LIKE '%%declare%%'
         """
```
Comment (harshsoni2024) [inline ingestion/src/metadata/ingestion/source/database/mssql/usage.py:27]: this `get_sqlalchemy_engine_dateformat` would be useful incase of server in different language.
these were recent PRs we did regarding the same.
[lineage](https://github.com/open-metadata/OpenMetadata/pull/17239/files)
[usage](https://github.com/open-metadata/OpenMetadata/pull/17341/files)
removing timezone form datetime object is alright but we need to convert same string in appropriate format.

**Review by ayush-shah — COMMENTED** [review]
Can we take care of lineage.py having similar changes? Thanks

File: ingestion/src/metadata/ingestion/source/database/mssql/lineage.py [hunk 2]
```diff
@@ -26,10 +26,6 @@
 
 
 class MssqlLineageSource(MssqlQueryParserSource, LineageSource):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.start = self.start.replace(tzinfo=None)
-        self.end = self.end.replace(tzinfo=None)
```
Comment (SumanMaharana) [inline ingestion/src/metadata/ingestion/source/database/mssql/lineage.py:32]: `.replace(tzinfo=None)` is handled in `helpers.py` itself so we don't need it here

File: ingestion/src/metadata/utils/helpers.py [hunk 3]
```diff
@@ -123,7 +123,7 @@ def get_start_and_end(duration: int = 0) -> Tuple[datetime, datetime]:
     Method to return start and end time based on duration
     """
 
-    today = datetime.now(timezone.utc)
+    today = datetime.now(timezone.utc).replace(tzinfo=None)
```
Comment (pmbrull) [inline ingestion/src/metadata/utils/helpers.py:126]: why pass the timezone and then remove it instead of `datetime.now()` without any specification?

Comment (SumanMaharana) [reply ingestion/src/metadata/utils/helpers.py:126]: just using `datetime.now()` will give the local time which can be different from the UTC time. to keep it standard we pass the timezone as `utc`

**Review by pmbrull — APPROVED** [review]

**Pull request merged**